src/_xoConnection.js

```diff
@@ -123,21 +131,50 @@ class XoConnection extends Xo {
     return id;
   }
 
+  async createResources(resourcesToCreate, disposer) {
+    const resources = {};
+    for (const typeOfResources in resourcesToCreate) {
+      for (const resource in resourcesToCreate[typeOfResources]) {
+        const { id } = await this.call(
+          METHOD[typeOfResources].create,
+          resourcesToCreate[typeOfResources][resource]
```

Comment:
What if we want to add a special property for a method?

Reply:
A special property? But, if we want to add a property, it can be added in the `resourcesToCreate[typeOfResources][resource]`, right?

Reply:
For all create methods ? 'remotes, servers, .....`?